src/types/src/atem.ts

```diff
@@ -114,32 +128,49 @@ export interface TimelineObjAtemME extends TimelineObjAtemBase {
 		deviceType: DeviceType.ATEM
 		type: TimelineContentTypeAtem.ME
 		me: {
+			/** Must be used with transition property, sets input to transition to */
 			input?: number,
 			transition?: AtemTransitionStyle,
 
-			// programInput?: number; // programInput exists, bu I don't think we should use it /Nyman
+			/** To change previewInput you must use programInput to set program directly */
```

Comment:
Using previewInput implies you can only change the program via programInput and not via input + transititon. In your current wording it seems like you can only change previewInput if you defined programInput which is not true. PR looks good otherwise.

Reply:
Good point, updated wording: https://github.com/olzzon/tv-automation-state-timeline-resolver/commit/e4b819e09d8a7bca89b0c6ac8aabe2190427d86d